### ISSUE TITLE
Color Component: No color contrast warning for link color against background color

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#38222]
### Description

Currently, in 5.9 there is not a color warning for the link color when compared against the background color. There is one in place for text color against background color and the same needs to exists for link color. 

### Step-by-step reproduction instructions

1. Add a paragraph block/write some text.
2. Make the background black in block settings. 
3. Make link color black. 
4. See no accessibility warning.
5. Make text color black, see accessibility warning. 

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/26996883/151054138-f8071e72-5737-4077-8e02-6b254089a810.mov

### Environment info

- WordPress 5.9
- No Gutenberg
- TT2
- Chrome
- MacOS

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ramonjd** [comment]
I think the merged PR https://github.com/WordPress/gutenberg/pull/38100 might close this one too? Reported previously in https://github.com/WordPress/gutenberg/issues/24595

**Comment by annezazu** [comment]
Sadly, no. I just tested with 5.9 and GB 12.5.3. Still no warning 😢

**Comment by ramonjd** [comment]
> Sadly, no. I just tested with 5.9 and GB 12.5.3. Still no warning 😢

Ah okay, the PR was merged today so it should be available on the next Gutenberg release.

**Comment by annezazu** [comment]
Seems fixed!

<img width="1286" alt="Screen Shot 2022-03-07 at 3 08 35 PM" src="https://user-images.githubusercontent.com/26996883/157110090-9fb9e2fa-6f2a-40dd-a4c3-b85776274130.png">
